pkg/bmt/benchmark_test.go

```diff
@@ -14,41 +14,42 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-//
-func BenchmarkBMT(t *testing.B) {
+func BenchmarkBMT(b *testing.B) {
 	for size := 4096; size >= 128; size /= 2 {
-		t.Run(fmt.Sprintf("%v_size_%v", "SHA3", size), func(t *testing.B) {
-			benchmarkSHA3(t, size)
+		b.Run(fmt.Sprintf("%v_size_%v", "SHA3", size), func(t *testing.B) {
+			benchmarkSHA3(b, size)
```

Comment:
`t` is passed as argument here. So these should be passed in to the benchmarkXXX functions.

Reply:
interestingly, this case was not reported by lint :+1: